foundations/installations/installations.md

```diff
@@ -261,14 +256,82 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
+</div>
+
+### Step 1: Installations
+
+#### Step 1.1: WSL2 and Virtual Machine Platform
+
+In order to make WSL2 work we'll first need to ensure both `Windows Subsystem for Linux` and `Virtual Machine Platform` are enabled on your system.
+
+- Search for Control Panel in your applications, and open it
+- Select **Programs**
+- Select **Turn Windows Features on or off** (under the Programs and Features header)
+- Ensure both `Windows Subsystem for Linux` and `Virtual Machine Platform` are toggled on, and if not toggle them on.
+- When prompted, restart your computer
+
+#### Step 1.2: Installing WSL2
+
+- Open PowerShell in administrator mode by searching for it in your applications, right clicking the top option, and then selecting run as administrator. You might get a prompt asking if you want to allow Windows Powershell to make changes to your device: click yes.
+- Enter the following command
+
+~~~powershell
+wsl --install
+~~~
+
+- After a few minutes you'll be prompted to reboot your computer; do so.
+- You should see an open Powershell window, prompting you to enter a username and a password. Your username should be lowercase, but can otherwise be whatever suits you. You'll also need to enter a new password.
+- When entering your password you might notice that you aren't seeing any visual feedback. This is a standard security feature in Linux, and will also happen in all future cases where you need to enter some form of password. Rest assured that you are actually inputting something.
+
+#### Step 1.3.1: Install Windows Terminal (Windows 10 only)
+
+Windows Terminal is an terminal application that lets you more easily customize and run terminals, as well as supporting multiple tabs which can each run their own different terminals.
+
+- [Install Window's Terminal](https://learn.microsoft.com/en-us/windows/terminal/install) by using the direct install option.
+
+#### Step 1.3.2: Setting WSL2 as default (Optional)
+
+Unless you are regularly using other terminals on your computer we recommend that you set WSL as the default terminal program when you open Windows Terminal.
+
+- Open Windows Terminal by searching for terminal in your applications.
+- Click the dropdown next to the new tab button (at the top of the windows), and select Settings.
+- You should see a Default Profile option with a dropdown next to it.
+- In the dropdown select Ubuntu
+- Click save at the bottom of the page
+
+### Step 2 Opening WSL2
+
+On Windows there are three primary ways to open WSL2.
+
+- If you set Windows Terminal to open a Ubuntu terminal by default you can start a new WSL2 session by simply opening the terminal app.
+- Otherwise you can open Windows Terminal, click the dropdown next to the new tab button (at the top of the windows), and select Ubuntu.
+- If you search for Ubuntu in the application search bar you should see a application titled Ubuntu; simply open it to start a new terminal session.
```

Comment:
```suggestion
- If you search for Ubuntu in the application search bar you should see a application titled Ubuntu; open it to start a new terminal session.
```

removed some mentions of "simply" since it is apparently used 11 times in this document and I started to notice it 😆